test/karma/helpers/feedbackViews.js

```diff
@@ -205,7 +205,7 @@ var ObjectFeedbackView = TorsoFormView.extend({
     }
   ],
 
-  template: '<input name="foo[abc]" data-model="foo[abc]" data-feedback="foo[abc]" /><input name="foo[def]" data-model="foo[def]" data-feedback="foo[def]" />',
+  template: '<input name="foo[abc]" data-model="foo.abc" data-feedback="foo[abc]" /><input name="foo[def]" data-model="foo.def" data-feedback="foo[def]" />',
```

Comment:
it's a bummer that the attributes are written differently. the feedback can't do foo.abc, right?

Reply:
I don't think it can...